airbyte-webapp/src/components/connection/CatalogTree/next/StreamDetailsPanel/StreamFieldsTable/StreamFieldsTable.tsx

```diff
@@ -0,0 +1,197 @@
+import { createColumnHelper } from "@tanstack/react-table";
+import React, { useMemo } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { ArrowRightIcon } from "components/icons/ArrowRightIcon";
+import { CheckBox } from "components/ui/CheckBox";
+import { NextTable } from "components/ui/NextTable";
+import { RadioButton } from "components/ui/RadioButton";
+
+import { SyncSchemaField, SyncSchemaFieldObject } from "core/domain/catalog";
+import { AirbyteStreamConfiguration } from "core/request/AirbyteClient";
+import { useConnectionFormService } from "hooks/services/ConnectionForm/ConnectionFormService";
+import { useDestinationDefinition } from "services/connector/DestinationDefinitionService";
+import { useSourceDefinition } from "services/connector/SourceDefinitionService";
+import { equal } from "utils/objects";
+import { getDataType } from "utils/useTranslateDataType";
+
+import { pathDisplayName } from "../../../PathPopout";
```

Comment:
The relative imports need to be cleaned up

Reply:
fixed